### PR TITLE
Translation of the blog's home page

### PR DESCRIPTION
**Pull request by khabzox**
I have just translated the home page of the blog and I am still working on completing the translation of the entire blog

[Preview](https://ar-react-dev-git-fork-khabzox-main-fbopensource.vercel.app/blog)

### CODE REVIEW

File: src/content/blog/index.md [hunk 1]
```diff
@@ -1,99 +1,106 @@
...
+2020 كان عامًا طويلًا. ومع اقترابه من نهايته، أردنا مشاركة تحديث خاص للعطلة حول بحثنا في مكونات خادم ريأكت بحجم صفري. لتقديم مكونات خادم ريأكت، قمنا بتحضير محاضرة وعرض توضيحي. إذا أردت، يمكنك التحقق منها خلال العطلات، أو لاحقًا عندما يعود العمل إلى وتيرته الطبيعية في العام الجديد...
+
 </BlogCard>
 
 </div>
 
 ---
 
-### All release notes {/*all-release-notes*/}
+### جميع ملاحظات الإصدار {/*all-release-notes*/}
```
Comment (AhmedBaset) [inline src/content/blog/index.md:96]:
```suggestion
### جميع ملاحظات الإصدارات {/*all-release-notes*/}
```

File: src/content/blog/index.md [hunk 2]
```diff
@@ -1,99 +1,106 @@
...
 
-Not every React release deserves its own blog post, but you can find a detailed changelog for every release in the [`CHANGELOG.md`](https://github.com/facebook/react/blob/main/CHANGELOG.md) file in the React repository, as well as on the [Releases](https://github.com/facebook/react/releases) page.
+ليس كل إصدار ريأكت يستحق منشور مدونة خاص به، ولكن يمكنك العثور على سجل تغيير مفصل لكل إصدار في ملف [`CHANGELOG.md`](https://github.com/facebook/react/blob/main/CHANGELOG.md) في مستودع ريأكت، وكذلك في صفحة [الإصدارات](https://github.com/facebook/react/releases).
```
Comment (AhmedBaset) [inline src/content/blog/index.md:98]:
```suggestion
ليس كل إصدار React يستحق منشور مدونة خاص به، ولكن يمكنك العثور على سجل تغيير مفصل لكل إصدار في ملف [`CHANGELOG.md`](https://github.com/facebook/react/blob/main/CHANGELOG.md) في مستودع React، وكذلك في صفحة [الإصدارات](https://github.com/facebook/react/releases).
```

**Review by AhmedBaset — CHANGES_REQUESTED** [review]
Thanks for opening this

Mostly, it looks good. I proposed some changes, please have a look

**Comment by khabzox** [comment]
I've made all the changes

[Preview](https://ar-react-dev-git-fork-khabzox-main-fbopensource.vercel.app/blog)

**Review by AhmedBaset — APPROVED** [review]
Thank you

**Pull request merged**